packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -659,6 +740,11 @@ describe('Table', () => {
             column1.sortDirection = TableColumnSortDirection.none;
             await waitForUpdatesAsync();
 
+            // Reopen the menu so we can verify the slots (changing sort order causes a row recycle which closes action menus)
+            toggleListener = createEventListener(element, 'action-menu-toggle');
```

Comment:
Why does this need to be re-created (just curious)?

Reply:
It's not super evident from the name, but `createEventListener` removes the event listener after the event is fired and handled once (i.e. is only meant to be used once). Since the `action-menu-toggle` event fires when the action menu is shown earlier in the test, we need to recreate it to listen for it the 2nd time.
Side note, it might make sense to encapsulate that event listening code inside `clickCellActionMenu`, as the code that opens action menus isn't really complete until after `action-menu-toggle` fires (I had issues with that in another test).